cypress/e2e/tests/migration/migration-waves/field_validation.test.ts

```diff
@@ -43,8 +51,8 @@ describe(["@tier1"], "Migration Waves Validations", () => {
         MigrationWave.openNewForm();
         cy.get(MigrationWaveView.submitButton).should("be.disabled");
 
-        validateTooShortInput(MigrationWaveView.nameInput, "body");
-        validateTooLongInput(MigrationWaveView.nameInput);
+        MigrationWave.validateWrongInput(MigrationWaveView.nameInput, "body");
+        MigrationWave.validateWrongInput(MigrationWaveView.nameInput);
```

Comment:
A modification of the `validateTooShortInput` method would work better, just adding a new parameter called `message` so the default message can be overrided. This will allow future use of the method for other components in the app.

Reply:
Same for the `validateTooLongInput` method

Reply:
Yeah that can be done .